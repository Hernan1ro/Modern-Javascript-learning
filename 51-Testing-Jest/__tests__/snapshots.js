const cliente = {
  nombre: "Hernan d",
  balance: 500,
  tipo: "Premium",
};

describe("Testing al cliente", function () {
  test("Es Hernan David", function () {
    expect(cliente).toMatchSnapshot();
  });
});
