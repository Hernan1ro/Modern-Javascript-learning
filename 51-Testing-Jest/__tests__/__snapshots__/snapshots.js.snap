// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Hernan David 1`] = `
Object {
  "balance": 500,
  "nombre": "Hernan d",
  "tipo": "Premium",
}
`;
